src/com/cybozu/labs/langdetect/Command.java

```diff
@@ -181,9 +190,17 @@ private void generateProfileFromText() {
         FileOutputStream os = null;
         try {
             LangProfile profile = GenProfile.loadFromText(lang, file);
-            profile.omitLessFreq();
-
             File profile_path = new File(lang);
+            if(get("update") != null){
+                try {
+                    FileInputStream is = new FileInputStream(profile_path);
+                    LangProfile old_profile = JSON.decode(is, LangProfile.class);
+                    profile.merge(old_profile);
+                }catch(Exception e){}

```

Comment:
I feel like I had a reason for why it was silent but no longer recall. I can add something.